ds3-sdk/src/test/java/com/spectralogic/ds3client/helpers/RequestMatchers.java

```diff
@@ -23,16 +23,15 @@
 import com.spectralogic.ds3client.commands.spectrads3.GetJobChunksReadyForClientProcessingSpectraS3Request;
 import com.spectralogic.ds3client.models.JobChunkClientProcessingOrderGuarantee;
 import org.apache.commons.io.IOUtils;
-import org.hamcrest.Description;
-import org.hamcrest.TypeSafeMatcher;
-import org.mockito.ArgumentMatcher;
+import org.hamcrest.*;
```

Comment:
Can we avoid the * import?

Reply:
Yep - I caught a few more of them that were added.